src/core/apiLayer/vacancyService.ts

```diff
@@ -0,0 +1,51 @@
+import qs from "query-string";
+
+const ServiceBaseUrl = "http://localhost:8082";
+// @FIXME: should be pulled from account level
```

Comment:
For demo, let's use DHL as company id. They've got 41k jobs :)
The id for that is `c5c6e9e5-ded0-4b41-9214-a874c5a6d7fb`

Reply:
Done.

https://github.com/vonq/saleor-storefront/blob/cf8a7430cb0e64c90f44059827be4009326d849d/src/constants.ts#L13-L14